src/screens/ExplainSheet.js

```diff
@@ -116,6 +116,13 @@ export const explainers = {
     readMoreLink: 'https://polygon.technology/',
     extraHeight: 120,
   },
+  wallet_connect: {
```

Comment:
can we change the name of this to failed_wc_connection or something more semantic?

Reply:
changed